packages/markdown/src/components/MarkdownPreview.tsx

```diff
@@ -197,13 +200,23 @@ const MarkdownPreview = React.memo((props: MarkdownPreviewProps) => {
   const className = cx(
     props.minHeight !== undefined ? css({ minHeight: props.minHeight }) : undefined,
     props.mode === 'default' ? styles.framed : styles.zen,
-    props.direction === 'rtl' ? styles.rtl : undefined
+    props.direction === 'rtl' ? styles.rtl : undefined,
   );
 
-  // See the list of allowed Tags here:
-  // https://github.com/cure53/DOMPurify/blob/main/src/tags.js#L3-L121
   const cleanHTML = React.useMemo(() => {
-    return replaceMailtoAmp(DOMPurify.sanitize(props.value));
+    try {
+      const processor = unified()
+        .use(rehypeParse, { fragment: true })
+        .use(rehypeSanitize)
```

Comment:
I think you can pass `rehypeSanitize` to the `rehypePlugins` array below & `props.value` instead of `cleanHTML`, no?

Reply:
You're right! 💯